store/ipfs.go

```diff
@@ -0,0 +1,62 @@
+package store
```

Comment:
How about making it a separate package `ipfs`?

Reply:
I put the `ipfs` under the `store` package because I think that `ipfs`'s role is also used as `store`. But in future, some file or method that is associated with `ipfs` will be generated. So I'll make `ipfs` package.